engines/bastion/app/views/bastion/layouts/assets.html.erb

```diff
@@ -10,11 +10,13 @@
 <% end %>
 
 <% content_for(:javascripts) do %>
-  <%= javascript_include_tag 'bastion/bastion' %>
+
+  <%=  javascript_tag("window.tfm.tools.loadPluginModule('/webpack/katello','katello','./common_index');".html_safe) %>
```

Comment:
This looks very similar to what `js_tags_for(requested_plugins)` already does for `katello`, except that this calls it for `common_index` and the other does it for `index`.

I'm trying to understand what `common_index.js` even does. I don't see anything that includes it and it only requires `ngreact`. If you don't have this, what happens? If it's needed, can it just be moved into katello's `index.js`?

Reply:
It loads angular, and since angular is not needed in all of Katello I think we shouldn't move it to index.
Without angular pages are just empty.

Reply:
That makes sense. Thanks for explaining.